dev/com.ibm.ws.security.social/src/com/ibm/ws/security/social/tai/TAIWebUtils.java

```diff
@@ -189,7 +189,7 @@ String getBearerTokenFromCustomHeaderSegments(HttpServletRequest req, String hea
         } catch (Exception e) {
             //can be ignored
             if (tc.isDebugEnabled()) {
-                Tr.debug(tc, "Fail to read Header Segments:", e);
+                Tr.debug(tc, "Fail to read Header Segments:", e.getMessage());
```

Comment:
This one could have stayed as just `e`, but it's a debug statement so isn't a big deal.